src/pyedb/dotnet/edb_core/edb_data/layer_data.py

```diff
@@ -794,13 +794,14 @@ def properties(self):
         data["thickness"] = self._edb_object.GetThicknessValue().ToString()
         data["color"] = self.color
 
-        etch_power_ground_nets = int(self._edb_layer.GetEtchNetClass())
-        etch_power_ground_nets = False if etch_power_ground_nets else True
         data["etching"] = {
             "factor": self._edb_layer.GetEtchFactor().ToString(),
             "enabled": self._edb_layer.IsEtchFactorEnabled(),
-            "etch_power_ground_nets": etch_power_ground_nets,
         }
+        if self._pedb.edbversion >= "2024.2":
```

Comment:
```suggestion
        if float(self._pedb.edbversion) >= 2024.2:
```

Reply:
@svandenb-dev this was discussed in pybreakfast. Comparing string is ok.